User signup/signin page created | backend functionality is yet to be …

pandacover opened the pull request:
…created | Changes to styles/globals.css

pandacover committed:
User signup/signin page created | backend functionality is yet to be created | Changes to styles/globals.css

TanishqSingla commented on components/Signuser/Signuser.tsx:
```diff
@@ -0,0 +1,83 @@
+  return (
+    <div className="w-3/12 h-fit-content mx-auto mt-20">
+      <h1 className="text-center text-4xl font-semibold uppercase">{type}</h1>
+      <form onSubmit={(e) => onSubmit(e)} className="h-fit-content w-full mt-8">
```
Refer material design on how to render forms and edit the design using those principles

pandacover replied:
Sure, would make a prototype for that later.

TanishqSingla commented on components/Signuser/Signuser.tsx:
```diff
@@ -0,0 +1,83 @@
+};
+
+const AuthForm = ({ type }: AuthProps) => {
+  const [userState, setUserState] = useState({
```
Are you sure we need one state to handle both the elements of the form, if we want to handle validations don't you think it would better to have 2 different states for both the fields.
Please provide your inputs

pandacover replied:
Having two states for different fields might be similar to what i did. A single state with fields as it's property also suffice since updation of both doesn't depend on each other.

TanishqSingla commented on components/Signuser/Signuser.tsx:
```diff
@@ -0,0 +1,83 @@
+            setUserState({ ...userState, password: ct.value })
+          }
+        />
+        {type === "signup" ? (
```
We can remove this, or either comment this

pandacover replied:
The layout didn't have anything about not adding a Signup component. Would do that before the next review.

TanishqSingla committed:
Removing package-lock.json

We already are running with yarn.lock, having package-lock.json will cause conflict in builds

TanishqSingla commented on styles/globals.css:
```diff
@@ -4,10 +4,23 @@
+    .surface-shadow {
+        @apply shadow-[0_1px_3px_rgba(0,0,0,0.12),0_1px_2px_rgba(0,0,0,0.24)];
+    }
+    .surface {
```
Please explain your intention here

pandacover replied:
It seems to be a work of the second party working in this repository. Thanks.

TanishqSingla reviewed (commented):
Please restore yarn.lock and look into the required changes

pandacover commented:
> Please restore yarn.lock and look into the required changes

Working with npm package control right now. Would do that before the next review.
